packages/functional-tests/tests/signUp/signUp.spec.ts

```diff
@@ -158,6 +159,27 @@ test.describe('signup here', () => {
     // The original tab should transition to the settings page w/ success
     // message.
     expect(await login.loginHeader()).toBe(true);
+
+    // Verify the account is in local storage and has a correct state
+    const currentAccountUid = await page.evaluate(() => {
```

Comment:
I'm kinda surprised there isn't a basic signup test, seems like it would fit in that better.

Reply:
I was also surprised. Maybe I just missed it though. Let me look again. If you know off hand of a better test, I'ld be happy to move these check.